tests/conftest.py

```diff
@@ -176,6 +178,48 @@ def bdb_node(bdb_host, bdb_port):
     return 'http://{host}:{port}'.format(host=bdb_host, port=bdb_port)
 
 
+"""Fixture to test with docker-compose up -d --scale bigchaindb=5"""
+
+
+@fixture
+def bdb_nodes(bdb_node):
+    return [bdb_node,
+            'bigchaindb-driver_bigchaindb_1:9984',
+            'bigchaindb-driver_bigchaindb_2:9984',
+            'bigchaindb-driver_bigchaindb_3:9984',
+            ]
+
+
+@fixture
+def bdb_nodes_headers(bdb_node):
+    return [
+            {'endpoint': bdb_node,
+             'headers': {'app_id': 'id'}},
+            {'endpoint': 'bigchaindb-driver_bigchaindb_1:9984',
+             'headers': {'app_id': 'id'}},
+            {'endpoint': 'bigchaindb-driver_bigchaindb_2:9984',
+             'headers': {'app_id': 'id'}},
+            {'endpoint': 'bigchaindb-driver_bigchaindb_3:9984',
+             'headers': {'app_id': 'id'}},
+            {'endpoint': 'bigchaindb-driver_bigchaindb_4:9984',
+             'headers': {'app_id': 'id'}},
+            {'endpoint': 'bigchaindb-driver_bigchaindb_5:9984',
+             'headers': {'app_id': 'id'}},
```

Comment:
would it make sense to have a fixture or var for the `header` since it doesn't change?